subsys/mgmt/mcumgr/Kconfig

```diff
@@ -101,6 +101,30 @@ config MGMT_MAX_DECODING_LEVELS
 	  size of cbor_nb_reader structure by zcbor_state_t size per
 	  one unit selected here.
 
+config MGMT_MCUMGR_MGMT_NOTIFICATION_HOOKS
```

Comment:
Sorry for asking to change this again, but I have agreed that the prefix `MGMT_` is too much, and `MCUMGR_` is distinctive enough.